BuildTools/FormatValidation/validate_format_linux.sh

```diff
@@ -3,6 +3,7 @@
 source ../../../DiligentCore/BuildTools/FormatValidation/validate_format_linux_implementation.sh
 
 validate_format ../../AssetLoader ../../Imgui ../../NativeApp/include  ../../NativeApp/src ../../TextureLoader ../../Tests \
+../../RenderStateNotationParser/include ../../RenderStateNotationParser/src ../../RenderStatePackager \
```

Comment:
Can you add ../../RenderStateNotationParser and exclude ../../RenderStateNotationParser/generated?
The reason is that if it happens that we add anything to RenderStateNotationParser, it will automatically be handled by the format validation script. Now it will requires manually adding new folders to the path list.